reference/3.0/Microsoft.PowerShell.Diagnostics/Get-Counter.md

```diff
@@ -9,273 +9,478 @@ title:  Get-Counter
+                          9031176192
+
+                          \\Computer01\memory\committed bytes :
+                          8242982912
+
+                          \\Computer01\memory\commit limit :
+                          19603333120
 ```
-The first command saves a counter path in the $Counter variable.
-PS C:\> $Counter = "\\SERVER01\Process(Idle)\% Processor Time"
 
-The second command uses the **Get-Counter** cmdlet to get one sample of the counter values. It saves the results in the $Data variable.
-PS C:\> $Data = Get-Counter $Counter
+`Get-Counter` uses the **ListSet** parameter to specify the **Memory** counter set. The command is
+enclosed in parentheses so that the **Paths** property returns each path as a string. The paths are
+stored in the `$MemCounters` variable. `Get-Counter` uses the **Counter** parameter to specify the
+counter paths in the `$MemCounters` variable.
+
+### Example 11: Display an object's property values
 
-The third command uses the Format-List cmdlet to display all the properties of the **CounterSamples** property of the sample set object as a list.
-PS C:\> $Data.CounterSamples | Format-List -Property *
+The property values in the **PerformanceCounterSample** object represent each data sample. You can
+use the properties of the **CounterSamples** object to examine, select, sort, and group the data.
 
-Path             : \\SERVER01\process(idle)\% processor time
+```powershell
+$Counter = "\\Server01\Process(Idle)\% Processor Time"
+$Data = Get-Counter $Counter
+$Data.CounterSamples | Format-List -Property *
+```
+
+```Output
+Path             : \\Server01\process(idle)\% processor time
 InstanceName     : idle
 CookedValue      : 198.467899571389
 RawValue         : 14329160321003
 SecondValue      : 128606459528326201
 MultipleCount    : 1
 CounterType      : Timer100Ns
-Timestamp        : 7/15/2008 6:39:12 PM
+Timestamp        : 6/19/2019 12:20:49
 Timestamp100NSec : 128606207528320000
 Status           : 0
 DefaultScale     : 0
 TimeBase         : 10000000
 ```
 
-This example shows the property values in the **PerformanceCounterSample** object that represents each data sample.
-You can use the properties of the CounterSamples object to examine, select, sort,  and group the data.
-### Example 10
-```
-PS C:\> Start-Job -ScriptBlock {Get-Counter -Counter "\LogicalDisk(_Total)\% Free Space" -MaxSamples 1000}
-```
+The counter path is stored in the `$Counter` variable. `Get-Counter` gets one sample of the counter
+values and stores the results in the `$Data` variable. The `$Data` variable uses the
+**CounterSamples** property to get the object's properties. The object is sent down the pipeline to
+`Format-List`. The **Property** parameter uses an asterisk (`*`) wildcard to select all the
+properties.
 
-The command runs a **Get-Counter** command as background job.
-For more information, see Start-Job.
+### Example 12: Use the CounterSamples property to display counters
```

Comment:
Example 11 and 12 are really duplicates. Remove Example 12 and renumber remaining.

Reply:
resolved.